lib/elastic_apm/subscriber.rb

```diff
@@ -54,14 +54,20 @@ def start(name, id, payload)
     end
     # rubocop:enable Metrics/MethodLength
 
-    def finish(_name, id, _payload)
+    def finish(name, id, payload)
       # debug "AS::Notification#finish:#{name}:#{id}"
       return unless (transaction = @agent.current_transaction)
 
       while (notification = transaction.notifications.pop)
         next unless notification.id == id
 
         if (span = notification.span)
+          _caller = caller
```

Comment:
I will probably change this to check if the normalizer can get a stacktrace so as not to waste calling `caller` unless I need to.